tests/index.test.js

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '../playwright/fixtures';
+
+test('Change greeting on button click', async ({ context, extensionId }) => {
```

Comment:
Should we mock the DOM objects here for the unit tests?

Reply:
Makes sense! Running the extension is better for e2e. Let me change this.

Reply:
I tried to mock, but it would still need a runtime to run the unit tests. Turns out we need the extension support. So let this be as is for now.